src/client/android/app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,12 @@
         </intent-filter>
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+        <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="@string/facebook_app_id"/>
```

Comment:
Didn't we have a limit on # of chars per each line? This is wrapping in github view. We should limit it to at least that # of chars..